device/project.godot

```diff
@@ -31,7 +31,12 @@ application/max_voice_volume=2
 application/save_data="res://globals/save_data.gd"
 application/speech_suffix=".spx"
 application/tooltip_lang_default="en"
+application/text_lang="en"
```

Comment:
Why is `tooltip_lang_default` and `text_lang` different settings? Are there any real use cases where a developer would want different language settings for them, or could `text_lang` be used for tooltips as well?

Reply:
Nevermind; I just saw the explanation in the documentation below.